neofs-local/README.md

```diff
@@ -5,22 +5,39 @@
 ## Keys folder
 *Contains pregenerated private keys for IR nodes*
 
-## Run environment
+## FAQ:
+
+### Configuration
+
+**Increase default upload file size limit:**
+
+By default, in .env you can find `NEOFS_OBJECT_MAX_PROCESSING_SIZE=300` (**300MB**),
```

Comment:
It is set in `.env` by `NEOFS_OBJECT_MAX_PROCESSING_SIZE` (**300MB** by default). You can override this value.